modules/tasks/tasks/src/main/resources/webapp/partials/modals/triggersModal.html

```diff
@@ -0,0 +1,77 @@
+<div class="tasks modal fade" id="triggersModal"  tabindex="-1" role="dialog">
```

Comment:
We should show how many pages there are total. We should also allow fast forward to last and first page. Would be also good to make the number an input that can be manually changed. In other words, make this more like jqgrid pagination.